### ISSUE TITLE
Missing sessions when running M2 on redis

### ISSUE THREAD

**Issue by wiardvanrij** [yireo/Yireo_DisableLog2#8]
I do lack information because I did not have the time to debug it, nonetheless I wanted to create an issue. 

What happend was that sessions did not persist for users on our shop having session storage in Redis. Sessions were made and actually stored in Redis but an user would nearly instantly lose his session (and therefor cart). Same for backend login - instant loss of the session.

**Comment by jissereitsma** [comment]
I'm not sure how to deal with this. You post this as an issue with the Yireo_DisableLog2 extension, but fundamental testing is required first from your side to proof that this problem is related to this extension. For instance, does this problem still occur when this extension is disabled? And does this issue occur with Redis only or also with file-based sessions?

I'm personally running DisableLog2 with Redis sessions on Magento 2.2.6 and Magento 2.3.0 and am experiencing no issues at all. Also, in the codebase, nothing related to sessions goes on, so it does not make sense to me either that such an issue could come to rise.

How come did you make the conclusion that your issue was related to Yireo DisableLog2? For instance, there could be something wrong with the Redis deamon, or the PHP environment, or another module? How come did you know for certain that the developers of the DisableLog2 extension should spend time on this instead?

**Comment by wiardvanrij** [comment]
Ok well; our devs made a small frontend fix (like 2 lines of css) and added this plugin. It seem to work on our staging but broke the session on production. I've spend time debugging it but could not find the root cause. After rollbacking to a release without this plugin everything was fine. 

I do know it lacks information but as I said I did not really have the time to debug it decently. I did however felt it was necessary to update you regarding our "findings". 

The assumption that it is related to Redis is perhaps a wrong one.  I only wanted to provide this information. 

Perhaps as extra information; it was a Commerce shop.
